lambda__lake_to_socrata.py

```diff
@@ -55,7 +55,7 @@ def lambda_handler(event, context):
 
     if event.get('source') == 'aws.events':
         overwrite = True
-        workingId = so_ingestor.create_new_draft()
```

Comment:
The var renamed in this file need to be lowercased (e.g. `workingId` -> `working_id`)